classifiers/query_check/communication_style_classifier/README.md

```diff
@@ -0,0 +1 @@
+Uses an intfloat/multilingual-e5-small model, which was finetuned on english and german examples of different question types. The model is hosted on Kern AIs on infrastructure and is meant to be used to classify text sequences by the labels action-seeking, fact-oriented, information-seeking or self-revealing.
```

Comment:
We could highlight `intfloat/multilingual-e5-small` and the labels e.g. `action-seeking`, using the grave accent `

Reply:
👍 Done